airbyte-integrations/connectors/destination-bigquery-denormalized/src/test-integration/java/io/airbyte/integrations/destination/bigquery/util/BigQueryDenormalizedTestDataUtils.java

```diff
@@ -61,6 +61,107 @@ public static JsonNode getSchema() {
                              """);
   }
 
+  public static JsonNode getAnyOfSchema() {
+    return Jsons.deserialize("""
+                             {
+                                "type":"object",
+                                "properties":{
+                                    "id":{
+                                        "type":[
+                                            "null",
+                                            "string"
+                                        ]
+                                    },
+                                    "name":{
+                                        "type":[
+                                            "null",
+                                            "string"
+                                        ]
+                                    },
+                                    "type":{
+                                        "type":[
+                                            "null",
+                                            "string"
+                                        ]
+                                    },
+                                    "email":{
+                                        "type":[
+                                            "null",
+                                            "string"
+                                        ]
+                                    },
+                                    "avatar":{
+                                        "type":[
+                                            "null",
+                                            "object"
+                                        ],
+                                        "properties":{
+                                            "image_url":{
+                                                "type":[
+                                                    "null",
+                                                    "string"
+                                                ]
+                                            }
+                                        },
+                                        "additionalProperties":false
+                                    },
+                                    "team_ids":{
+                                        "anyOf":[
+                                            {
+                                                "type":"array",
+                                                "items":{
+                                                    "type":"integer"
+                                                }
+                                            },
+                                            {
+                                                "type":"null"
+                                            }
+                                        ]
+                                    },
+                                    "admin_ids":{
+                                        "anyOf":[
```

Comment:
This test case only covers `anyOf`. Maybe add a new `allOf` field, or change this field to `allOf` to test that as well?

Reply:
I added a unit that also tests "allOf".